src/Util/RecursiveBlockIterator.php

```diff
@@ -48,6 +45,6 @@ public function getChildren(): self
 
     public function hasChildren(): bool
     {
-        return $this->current()->hasChildren();
+        return $this->current()->hasChild();
     }
```

Comment:
hasChildren is in the RecursiveArrayIterator class.

So I wonder Reverting the deprecation for hasChildren -> hasChild in the BlockInterface to use the same method name. WDYT @jordisala1991 ? Do you prefer hasChild or hasChildren ?

Reply:
which one is more correct? I think if it complicates things we have more things to improve before namings.